media/js/firefox/hello/index.js

```diff
@@ -119,12 +119,38 @@
         }
         // Hello exists in desktop version 35 and up
         else if (w.getFirefoxMasterVersion() >= 35) {
-            showFxFooterMessaging();
```

Comment:
Since `showFxFooterMessaging();` is now inside the promise, if it never fulfills (e.g. uitour doesn't work as expected), the fallback content doesn't show. We should make sure it degrades gracefully still